### PR TITLE
(port) primitive processor

### PR DESCRIPTION
**Pull request by boodaliboo**
original pr :https://github.com/Skyrat-SS13/Skyrat13/pull/3015 
adds a chopping block
for ghetto or primitive processing and cooking
:cl:  
rscadd: Added primitive processing
/:cl:

### CODE REVIEW

**Comment by Cartlord** [comment]
How could a chopping block - which is, presumably, unpowered - actually cook food?

**Comment by boodaliboo** [comment]
> How could a chopping block - which is, presumably, unpowered - actually cook food?

heat rocks

File: code/game/machinery/primitiveprocessor.dm [hunk 1]
```diff
@@ -0,0 +1,157 @@
+//here is the chopping block.
+//it can be created from 2 wooden planks
+//place this on a table or else you will build it in your hands, which places at your feet!
+/obj/item/chopping_block
+	name = "chopping block"
+	desc = "A primitive version that cavemen used to use for processing food. Yuck!"
```
Comment (Moggram) [inline code/game/machinery/primitiveprocessor.dm:6]: My god this grammar. Also fix the lines on 22 and 75,

"A primitive version that was used by cavemen for processing food. Yuck!"

**Comment by Marmio64** [comment]
i personally love to make tater tots with nothing but a block of wood and a knife

**Comment by Hopekz** [comment]
You guys are straight-up haters this idea is alright

There is no alternative at the moment for something like this and if someone wants more realism in ss13 I'm sure someone will follow up with something that looks like it can cook properly

**Comment by SomeguyManperson** [comment]
as much as I do want to have an alternative to a processor machine this code is terrible and I'm not sure how it got merged on skyrat

**Pull request closed without merging**